spring-kafka/src/test/java/org/springframework/kafka/annotation/EnableKafkaIntegrationTests.java

```diff
@@ -1159,6 +1159,7 @@ public RecordPassAllFilter manualFilter() {
 			return factory;
 		}
 
+		@SuppressWarnings("deprecation")
```

Comment:
Isn't there a way to avoid this with existing/new behavior?

I guess we will come back to the problem in this test when we are going to remove that `setAckOnError` eventually...

Reply:
Right - I didn't want to remove the test given that the feature is deprecated only; this test will be changed when we remove it.